core/modules/filters/wikiparserrules.js

```diff
@@ -16,10 +16,11 @@ Filter operator for returning the names of the wiki parser rules in this wiki
 Export our filter function
 */
 exports.wikiparserrules = function(source,operator,options) {
-	var results = [];
+	var results = [],
+		operand = operator.operand;
 	$tw.utils.each($tw.modules.types.wikirule,function(mod) {
 		var exp = mod.exports;
-		if(exp.types[operator.operand]) {
+		if(!operand || exp.types[operand]) {
```

Comment:
Why do you want to be able to use wikiparserrules without an operand? It looks like you're always calling it with a parameter in the rest of the commit.